### ISSUE TITLE
Hide "whats happening" also hides trends tab.

### ISSUE THREAD

**Issue by Nillx** [insin/control-panel-for-twitter#321]
Noticed that after clicking Hide "Whats happening." "You might like." option under Remove algorithmic content also hides the trends tab. This started happning recently.

**Comment by insin** [comment]
Do you mean the tabs on the Explore page? They're hidden by Remove algorithmic content > Hide Explore page contents and use it only for searching

**Comment by Nillx** [comment]
Nope, worldwide trends on the right 
Turned on:
![image](https://github.com/insin/control-panel-for-twitter/assets/34476875/88a7a0b7-378c-4aec-b6fb-a1e2bffca312)

Turned off
![image](https://github.com/insin/control-panel-for-twitter/assets/34476875/12b43a4f-6cad-4485-aaa4-8b970ff80096)

**Comment by Rocket3192** [comment]
Yep, it was following that support ticket https://github.com/insin/control-panel-for-twitter/issues/319#event-9561497207 , but it didn't fix anything lol we still can't hide the blue subscription without hiding the trends :(

**Comment by xmj** [comment]
An update:

To hide the trends tab in the current version, make sure you have

![image](https://github.com/user-attachments/assets/7562e7c8-2dc7-4015-bc8f-094c6775e074)

enabled.
